### ISSUE TITLE
Table of contents

### ISSUE THREAD

**Issue by Darkwolf1515** [anansi-project/rfcs#20]
Table of contents hasn't really been a thing in comic formats, some manga/comics I convert to cbz had a table of contents in their ebook format, would be great to be able to take them too into the cbz.

**Comment by gotson** [comment]
Can you describe what's a table of content, and how it is used?

**Comment by Darkwolf1515** [comment]
![8D50DCCE-5BA3-48E6-B245-0869D134B88E](https://user-images.githubusercontent.com/65789901/92435177-850e2680-f156-11ea-8dd3-28bbc28f362c.jpeg)

**Comment by gotson** [comment]
I suppose the idea behind would be to offer quick navigation?

**Comment by Darkwolf1515** [comment]
Yes, allow in this format to say "pg 25= chapter 2, pg 36= chapter 3, and so on.

**Comment by gotson** [comment]
We can add that in the data model, but i have doubts on the capacity of readers to integrate that 😓

**Comment by Darkwolf1515** [comment]
> 
> 
> We can add that in the data model, but i have doubts on the capacity of readers to integrate that 😓

Yeah, it'll be a big enough issue to get readers to adopt the format, let alone stuff like that. At least Komga could, if you think this is worth adding to the model.

**Comment by gotson** [comment]
Epub has it, but epub misses a lot of other things. If you have epubs, why concert to cbz in the first place?

**Comment by Darkwolf1515** [comment]
> 
> 
> Epub has it, but epub misses a lot of other things. If you have epubs, why concert to cbz in the first place?
To use in komga, of course.

**Comment by gotson** [comment]
Is that epub books or epub made of images only? The latter would work in Komga.

**Comment by Darkwolf1515** [comment]
> 
> 
> Is that epub books or epub made of images only? The latter would work in Komga.

Images only, regardless I dont think the table of contents would appear, would they?

**Comment by gotson** [comment]
No, but you wouldn't have lost it during conversion, and it would work with other readers.

**Comment by Darkwolf1515** [comment]
> 
> 
> No, but you wouldn't have lost it during conversion, and it would work with other readers.

Can always redownload it, but I have no intentions on reading my comics/manga anywhere but komga/panels on ios.

**Comment by lordwelch** [comment]
The ComicRack metadata does support bookmarks there is some discussion and links in anansi-project/comicinfo#13
